source/extensions/filters/http/ext_proc/client_impl.cc

```diff
@@ -75,12 +79,18 @@ void ExternalProcessorStreamImpl::onReceiveTrailingMetadata(Http::ResponseTraile
 void ExternalProcessorStreamImpl::onRemoteClose(Grpc::Status::GrpcStatus status,
                                                 const std::string& message) {
   ENVOY_LOG(debug, "gRPC stream closed remotely with status {}: {}", status, message);
-  callbacks_.logGrpcStreamInfo();
   stream_closed_ = true;
+
+  if (!callbacks_.has_value()) {
```

Comment:
Missing test (ideally unit) coverage.

Reply:
I tried unit tests: 
client_test : no ext_proc filter involved; so I can not configure ob11y mode.
filter_test: no remote server involved: server failure is mocked by `stream_callbacks_->onGrpcError(Grpc::Status::Internal)` where this `stream_callbacks_` is callback here that becomes invalid.

Thus it is probably easier to be tested in a integrated environment with filter+remote server. Integration test is added and confirmed coverage with logging